README.md

```diff
@@ -206,6 +206,9 @@ Then, install Haiku using pip:
 $ pip install git+https://github.com/deepmind/dm-haiku
 ```
 
+In order to run the RL `examples/` you will also need to install
+[bsuite](https://github.com/deepmind/bsuite).
```

Comment:
Would you mind adding a link to the requirements.txt file you added here instead? Something along the lines of:

```
Our examples rely on additional libraries (e.g. [bsuite](https://github.com/deepmind/bsuite)). You can install the full set of additional requirements using pip:

$ pip install -r haiku/examples/requirements.txt
```

Reply:
Good idea, done!